src/test/kotlin/uk/gov/justice/digital/hmpps/prisonregister/resource/GpResourceTest.kt

```diff
@@ -59,7 +59,7 @@ class GpResourceTest : IntegrationTest() {
     webTestClient.get().uri("/gp/prison/1234561234561")
       .exchange()
       .expectStatus().isBadRequest
-      .expectBody().json("prison_id_badrequest_getPrisonFromId".loadJson())
+      .expectBody().json("prison_id_badrequest_getPrisonFromIdForGP".loadJson())
```

Comment:
Needed a separate file as it was using the same json.  This isn't code that our team maintain, so left the json returned as previously expected